### PR TITLE
MGMT-9971: removing timeout from waiting for 2 master.

### PR DESCRIPTION
**Pull request by tsorya**
[MGMT-9971](https://issues.redhat.com//browse/MGMT-9971): removing timeout from waiting for 2 master.
Moving back to the logic where timeout is decision is taken only by
service

### CODE REVIEW

**Comment by filanov** [comment]
@tsorya lint issue `waitForMastersTimeout` is not used

File: src/installer/installer.go [hunk 1]
```diff
@@ -425,40 +424,28 @@ func (i *installer) workerWaitFor2ReadyMasters(ctx context.Context) error {
...
 	i.log.Info("Waiting for 2 ready masters")
 	i.UpdateHostInstallProgress(models.HostStageWaitingForControlPlane, "")
-	start := time.Now()
-	err := utils.WaitForPredicate(waitForMastersTimeout, generalWaitInterval, func() bool {
-		var callErr error
-		cluster, callErr = i.inventoryClient.GetCluster(ctx, false)
-		if callErr != nil {
-			i.log.WithError(callErr).Errorf("Getting cluster %s", i.ClusterID)
-			return false
+	_ = utils.WaitForPredicate(waitForeverTimeout, generalWaitInterval, func() bool {
```
Comment (ori-amizur) [inline src/installer/installer.go:427]: We need to bail out if the cluster or host is in error state.  Otherwise, it will loop forever

Comment (tsorya) [reply src/installer/installer.go:427]: we don't, service will send installation stop command in case cluster is in error

Comment (ori-amizur) [reply src/installer/installer.go:427]: /lgtm

**Comment by tsorya** [comment]
/retitle Bug 2078213: removing timeout from waiting for 2 master

**Pull request merged**